src/wizard/ionic/index.md

```diff
@@ -0,0 +1,134 @@
+---
+name: Capacitor
```

Comment:
Name should be Ionic instead?'
```suggestion
name: Ionic
```

I wonder if it's worth adding a page for Ionic on the root (similar to how we have Kotlin) where we just clarify we have Capacitor and Cordova and the preferred SDK is Capacitor now, etc.


Search now shows up some Cordova stuff on the top:
![image](https://user-images.githubusercontent.com/1633368/167924253-bb3df1c9-88f8-482b-8352-1ee87df14ebe.png)